public/app/features/dashboard-scene/panel-edit/PanelVizTypePicker.tsx

```diff
@@ -25,6 +26,13 @@ export function PanelVizTypePicker({ panel, data, onChange, onClose }: Props) {
   const styles = useStyles2(getStyles);
   const [searchQuery, setSearchQuery] = useState('');
 
+  const handleSearchChange = (value: string) => {
+    if (value) {
+      reportInteraction('dashboards_dspickermodal_clicked', { item: 'search', query: value });
```

Comment:
Need to pass search result count, but filtering happens inside the [child component.](https://github.com/grafana/grafana/pull/94890/files#diff-dd6934a79d19977e13188e9cc17ddbf74896145f176e8de9faa938a6d87c853aR94)